src/test/ui/entry-point/imported_main_conflict.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(imported_main)]
+//~^ ERROR `main` function not found in crate
+mod m1 { fn main() {} }
+mod m2 { fn main() {} }
```

Comment:
Both `main`s need to be `pub` or `pub(crate)`, otherwise the glob imports below don't import anything so we are not testing what we want to test.

Reply:
Indeed! This is an oversight. Arguely i think this code pattern should trigger a lint warning or something.